extensions/wikia/AdEngine/js/slot/service/megaAdUnitBuilder.js

```diff
@@ -10,16 +10,10 @@ define('ext.wikia.adEngine.slot.service.megaAdUnitBuilder', [
 	var dfpId = '5441',
 		megaSlots = [
 			'INVISIBLE_SKIN',
-			//ToDo ADEN-6864: Go global
-			//'BOTTOM_LEADERBOARD'
+			'BOTTOM_LEADERBOARD'
 		],
 		context;
 
-	//ToDo ADEN-6864: Go global
-	if (getContextTargeting().skin === 'mercury' || adContext.get('opts.isBLBMegaEnabled')) {
```

Comment:
I think [param in adContext `opts.isBLBMegaEnabled`](https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/context/adContext.js#L216) also can be removed

Reply:
We have follow up PR to merge after release+cache of this one:
https://github.com/Wikia/app/pull/15079/files